CMakeLists.txt

```diff
@@ -0,0 +1,571 @@
+# # CMake project for fdk-aac
+#
+# ## Basic usage
+#
+# 1. Download and install CMake >= 3.10 (CMake >= 3.15 is recommended)
+# 2. Clone fdk-aac repository
+# 3. In fdk-aac directory create and go to `build` subdirectory:
+#
+#    mkdir build
+#    cd build
+#
+# 4. Run cmake to configure project with desired build type (Release|Debug):
+#
+#    cmake .. -DCMAKE_BUILD_TYPE=Release
+#
+#    Note CMake configuration without CMAKE_BUILD_TYPE option will not set build
+#    type to some default and it is most likely is not what you want.
+#
+#    If you want to build DLL (default is static library), add BUILD_SHARED_LIBS=ON option:
+#
+#    #    cmake .. -DBUILD_SHARED_LIBS=ON
+#
+#    You can combine options as well:
+#
+#    cmake .. -DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS=ON
+#
+# 5. Run cmake to build project:
+#
+#    cmake --build . -c Release
+
+cmake_minimum_required(VERSION 3.10)
+
+# Policies
+
+## Enables CMAKE_MSVC_RUNTIME_LIBRARY option support for CMake >= 3.15
+## if you want to use a MSVC multi-threaded statically-linked runtime library
+## If you enable it, CMake will build fdk-acc.dll without external dependencies.
+##
+## Example usage:
+##
+## cmake .. -DCMAKE_MSVC_RUNTIME_LIBRARY="MultiThreaded$<$<CONFIG:Debug>:Debug>
+cmake_policy(SET CMP0091 NEW)
```

Comment:
This requires a `if(POLICY CMP0091)` on older versions of cmake.

Reply:
Oops, my fault.